### DISCUSSION TITLE
Form plugin - default ring shadow should be removed

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [tailwindlabs/tailwindcss#4227] [Ideas]
I think this default css on input tags is very annoying, and it's so difficult to remove.
Even with `!shadow-none !outline-none` it doesn't remove. The only way is` box-shadow: none !important `
<img width="622" alt="Schermata 2021-05-03 alle 14 06 28" src="https://user-images.githubusercontent.com/63153191/116874120-1156e780-ac19-11eb-8473-9ae4549b0462.png">

**Comment by adamwathan** [comment]
Use `focus:ring-0`.
